make/base_images.mk

```diff
@@ -1,12 +1,12 @@
 # +skip_license_check
 # autogenerated by hack/latest-base-images.sh
-STATIC_BASE_IMAGE_amd64 := gcr.io/distroless/static-debian12@sha256:fd92c3ad6367e38449ee4547ba87473d87a693d272d8670e326c585c29fa25f2
-STATIC_BASE_IMAGE_arm64 := gcr.io/distroless/static-debian12@sha256:a37f767b5baa75fe4e4e697a8d2f8e9389271f9c232124a3ad1019c97bae9708
-STATIC_BASE_IMAGE_s390x := gcr.io/distroless/static-debian12@sha256:2c735f980a7414418dc07c3dc9f7e235d17af19d64dbc050d6a6ae379c624885
-STATIC_BASE_IMAGE_arm   := gcr.io/distroless/static-debian12@sha256:0c0ff35c742c9bf48381051eb505eae04de1d0aabd1fbd55c76aa4e315df7451
-STATIC_BASE_IMAGE_ppc64le := gcr.io/distroless/static-debian12@sha256:3caba0be2811fc18748c09fa026419d34e92380a599234bf26c33fed00e627c2
-DYNAMIC_BASE_IMAGE_amd64 := gcr.io/distroless/base-debian12@sha256:ad04bf079b9ed668d38fe2138cfe575847795985097b38a400f4ef1ff69a561a
-DYNAMIC_BASE_IMAGE_arm64 := gcr.io/distroless/base-debian12@sha256:7d7d1369ba93fdbb34c0db570729047cbf8a1737c292eadde26a1d5d899a35e5
-DYNAMIC_BASE_IMAGE_s390x := gcr.io/distroless/base-debian12@sha256:a376bb78bf9eab614768f089ca3aa7aead71264f3599fd065a0faa25fc1deeb9
-DYNAMIC_BASE_IMAGE_arm   := gcr.io/distroless/base-debian12@sha256:4a4167eee2be030773bd3ab04c08680717ecc8683977ea2a5f89ac6900812b1d
-DYNAMIC_BASE_IMAGE_ppc64le := gcr.io/distroless/base-debian12@sha256:6641dfae1307e598dc70af6fe23f10daed015c02dd83255eabeafc104e9c32ce
+STATIC_BASE_IMAGE_amd64 := gcr.io/distroless/static-debian12@sha256:e14f4c00d388743628c674d88e44ac51895d7bc074a37e805eaf141cdbfa5ff1
+STATIC_BASE_IMAGE_arm64 := gcr.io/distroless/static-debian12@sha256:338640f5abf534cbee57abc6a7bf8d4782e1601b32ada5724e5329c2ac65706e
+STATIC_BASE_IMAGE_s390x := gcr.io/distroless/static-debian12@sha256:6666fd8941e5e65c869b16f4361fea0b717817a8d59d109a29e41389375d999e
+STATIC_BASE_IMAGE_arm   := gcr.io/distroless/static-debian12@sha256:548922a3f8d058d0774a143b4068d5ec818fa85c56a57630b53935ebd6a78c77
+STATIC_BASE_IMAGE_ppc64le := gcr.io/distroless/static-debian12@sha256:2efa371a664c58c53df39b89116b11f51afdd4fed54fd42b70c1c877d9229d01
+DYNAMIC_BASE_IMAGE_amd64 := gcr.io/distroless/base-debian12@sha256:d65201b3fe8eb56abcec05aec99acec815e40b10b63b854ce02bec4575f64526
+DYNAMIC_BASE_IMAGE_arm64 := gcr.io/distroless/base-debian12@sha256:72b7cd7a57701b69261c3a8900741bd389dfa2c8c676fcb6ec6b3c1c30592280
+DYNAMIC_BASE_IMAGE_s390x := gcr.io/distroless/base-debian12@sha256:a6e2d61fd3859f55ed1937b799bc07d846f1f07b4e1698b6807ea7e6649eb19e
+DYNAMIC_BASE_IMAGE_arm   := gcr.io/distroless/base-debian12@sha256:78b58c33698d5f1db7e16587729c66db76734758e5ba12365e539348846a5007
+DYNAMIC_BASE_IMAGE_ppc64le := gcr.io/distroless/base-debian12@sha256:63e77fa01b1fbfdec8facfca9cbd64f10e14ff0d2337321db9a30f5758dec4fa
```

Comment:
These were last updated in https://github.com/cert-manager/cert-manager/pull/7529
 * https://github.com/cert-manager/cert-manager/pull/7529